kadi/commands/states.py

```diff
@@ -1855,7 +1958,7 @@ def get_states(
 
         # Process the transition.
         for key, value in transition.items():
-            if inspect.ismethod(value):
+            if callable(value):
```

Comment:
What is expected if anything by this change?

Reply:
No change to existing code and methods. But without the change then the `functools.partial` wrapper in `set_transitions()` above is rejected since it is no longer a method according to `ismethod()`. That causes the fids state stuff to fail.

Reply:
Thanks!  I'd read the paragraph in the Description that hints at this but hadn't carried through to understanding this change requirement for the if statement.